tests/runtests/reporters-home-config/runtest.sh

```diff
@@ -0,0 +1,179 @@
+#
+#   runtest.sh of reporter-bugzilla, reporter-rhtsupport, reporter-mantisbt
+#   Description: Verify their ability to read configuration from current user's home
+#   Author: Julius Milan <jmilan@redhat.com>
+#
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#
+#   Copyright (c) 2017 Red Hat, Inc. All rights reserved.
+#
+#   This program is free software: you can redistribute it and/or
+#   modify it under the terms of the GNU General Public License as
+#   published by the Free Software Foundation, either version 3 of
+#   the License, or (at your option) any later version.
+#
+#   This program is distributed in the hope that it will be
+#   useful, but WITHOUT ANY WARRANTY; without even the implied
+#   warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+#   PURPOSE.  See the GNU General Public License for more details.
+#
+#   You should have received a copy of the GNU General Public License
+#   along with this program. If not, see http://www.gnu.org/licenses/.
+#
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+. /usr/share/beakerlib/beakerlib.sh
+. ../aux/lib.sh
+
+TEST="reporters-home-config"
+PACKAGE="abrt"
+
+TEST_DIR="."
+
+rlJournalStart
+    rlPhaseStartSetup
+        check_prior_crashes
+
+        # unset global configuration
+        augtool clear /files/etc/libreport/plugins/bugzilla.conf/Login
+        augtool clear /files/etc/libreport/plugins/bugzilla.conf/Password
+        augtool clear /files/etc/libreport/plugins/rhtsupport.conf/Login
+        augtool clear /files/etc/libreport/plugins/rhtsupport.conf/Password
+        augtool clear /files/etc/libreport/plugins/mantisbt.conf/Login
+        augtool clear /files/etc/libreport/plugins/mantisbt.conf/Password
+
+        # ensure local config exists
+        mkdir -p  ~/.config/libreport/
+        touch ~/.config/libreport/bugzilla.conf
+        touch ~/.config/libreport/rhtsupport.conf
+        touch ~/.config/libreport/mantisbt.conf
+
+        TmpDir=$(mktemp -d)
+        cp -v $TEST_DIR/expect $TmpDir/expect
+        pushd "$TmpDir"
+    rlPhaseEnd
+
+    rlPhaseStartTest "Check reporter-bugzilla, without config in HOME"
+        prepare
+        generate_crash
+        wait_for_hooks
+        get_crash_path
+
+        # unset local configuration
+        rlRun "augtool clear /files/$HOME/.config/libreport/bugzilla.conf/Login" 0 "Unset local Login"
+        rlRun "augtool clear /files/$HOME/.config/libreport/bugzilla.conf/Password" 0 "Unset local Password"
+
+        rlRun "yes no | reporter-bugzilla -v -d $crash_PATH &> output" 1 "Try to report by reporter-bugzilla"
+
+        rlAssertGrep "Login is not provided by configuration. Please enter your Bugzilla login:" output
+        rlAssertGrep "Password is not provided by configuration. Please enter the password for" output
+
+        rlRun "abrt-cli rm $crash_PATH" 0 "Remove crash dir"
+        rm output
```

Comment:
Do not remove log files. It's better to gather all logs by `rlBundleLogs` for further analysis. In this case would be fine to have each log file with a unique name.